src/services/ledger/domain/legacy-entry-builder.ts

```diff
@@ -0,0 +1,247 @@
+import { AmountCalculator, WalletCurrency } from "@domain/shared"
```

Comment:
do we plan to remove this file? else, please rename

Reply:
Yes we plan on removing this file.